crates/fuel-core/src/schema/coins.rs

```diff
@@ -40,6 +40,7 @@ use fuel_core_types::{
     fuel_tx,
 };
 use itertools::Itertools;
+use tokio_stream::StreamExt;
```

Comment:
Any reason why we use `StreamExt` from Tokio here and from Futures in other places? The one from the Futures crate supports more functions if I recall correctly.

Reply:
`tokio_stream::StreamExt` provides another functionality that works with closures instead of futures.

Reply:
Alright, good to know :)